src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -163,6 +163,11 @@ trait MarathonConf
     true
   }
 
+  lazy val gracefulShutdownTimeout = opt[Long] (
```

Comment:
oof. Do we really need an option for this? Can we use an undocumented environment variable at first?

Reply:
Fair point.